packages/url_launcher/url_launcher/lib/url_launcher.dart

```diff
@@ -71,8 +71,12 @@ Future<bool> launch(
   Brightness? statusBarBrightness,
   String? webOnlyWindowName,
 }) async {
-  final Uri url = Uri.parse(urlString.trimLeft());
-  final bool isWebURL = url.scheme == 'http' || url.scheme == 'https';
+  bool isWebURL = false;
+  try {
+    final Uri url = Uri.parse(urlString.trimLeft());
+    isWebURL = url.scheme == 'http' || url.scheme == 'https';
+  } on FormatException catch (_) {}
```

Comment:
Actually, you can just use `Uri.tryParse` instead of this, and simplify the code. I forgot to see if there was a `try` version of this API before my last review comments.

```
  final Uri? url = Uri.tryParse(urlString.trimLeft());
  final bool isWebURL = url != null && (url.scheme == 'http' || url.scheme == 'https');
```